pypesto/profile/validation_intervals.py

```diff
@@ -0,0 +1,106 @@
+import logging
+from typing import Optional
+
+from ..engine import Engine
+from ..optimize import Optimizer, minimize
+from ..problem import Problem
+from ..result import Result
+from scipy.stats import chi2
+
+
+logger = logging.getLogger(__name__)
+
+
+def validation_profile_significance(
+        problem_full_data: Problem,
+        result_training_data: Result,
+        result_full_data: Optional[Result] = None,
+        n_starts: int = 1,
+        optimizer: Optional[Optimizer] = None,
+        engine: Engine = None,
+        lsq_objective: float = False
```

Comment:
```suggestion
        lsq_objective: float = False,
```
just playing flake8 ...